### PR TITLE
Parse data

### PR DESCRIPTION
**Pull request by manjymon**
- Parse data to jsons and dump in files.
- Added contractions are because most tweets use a unicode character (’) instead of ' for contractions
- Fix unicode removal?

### CODE REVIEW

File: hate-detect/common_ngrams_finder.py [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import nltk
+#nltk.download('words')
+#nltk.download('averaged_perceptron_tagger')
+#nltk.download('universal_tagset')
```
Comment (manjymon) [inline hate-detect/common_ngrams_finder.py]: Where should we put these?

Comment (NickGeorgiev) [reply hate-detect/common_ngrams_finder.py]: I think we should move them to a config file or something?

File: hate-detect/extract_features.py [hunk 2]
```diff
@@ -13,16 +13,16 @@
 COMMON_POS_BIGRAMS = set()
 COMMON_POS_TRIGRAMS = set()
 
-with open('unigrams.json', 'r') as file:
+with open('../json/sunigrams.json', 'r') as file:
```
Comment (TheRedLady) [inline hate-detect/extract_features.py]: Is this a typo?

File: hate-detect/load_data.py [hunk 3]
```diff
@@ -1,44 +1,25 @@
...
-    for line in csv_file:
-        if len(line) > 0:
-            tweet = line[0]
-            tweet = initial_text_clean_up(tweet)
-            if len(tweet.split()) > 3:
-                data.append(tweet)
-    # remove duplicates
-    data = list(set(data))
-    return data
+def parse_CSV(csv_path):
```
Comment (NickGeorgiev) [inline hate-detect/load_data.py:8]: csv?

**Review by NickGeorgiev — CHANGES_REQUESTED** [review]

**Pull request merged**